packages/actions/src/helpers/s3.ts

```diff
@@ -0,0 +1,320 @@
+ * @returns <boolean> 
+ */
+export const createS3Bucket = async (
+    functions: Functions, 
+    bucketName: string): Promise<boolean> => {
+    const cf = httpsCallable(functions, 'createBucket')
+    // Call createBucket() Cloud Function.
+    const response: any = await cf({
+        bucketName
+    })
+
+    // Return true if exists, otherwise false.
+    return response.data
+}
+
+/**
+ * Check if an object exists in a given AWS S3 bucket.
+ * @param functions <Functions> - the cloud functions.
+ * @param bucketName <string> - the name of the AWS S3 bucket.
+ * @param objectKey <string> - the identifier of the object.
+ * @returns Promise<string> - true if the object exists, otherwise false.
+ */
+export const objectExist = async (
+    functions: Functions,
+    bucketName: string,
+    objectKey: string
+): Promise<boolean> => {
+    const cf = httpsCallable(functions, 'checkIfObjectExist')
+    // Call checkIfObjectExist() Cloud Function.
+    const response: any = await cf({
+        bucketName,
+        objectKey
+    })
+
+    // Return true if exists, otherwise false.
+    return response.data
+}
+
+/**
+ * Initiate the multi part upload in AWS S3 Bucket for a large object.
+ * @param cf <HttpsCallable<unknown, unknown>> - the corresponding cloud function.
+ * @param bucketName <string> - the name of the AWS S3 bucket.
+ * @param objectKey <string> - the identifier of the object.
+ * @param ceremonyId <string> - the identifier of the ceremony.
+ * @returns Promise<string> - the Upload ID reference.
+*/
+const openMultiPartUpload = async (
+    cf: HttpsCallable<unknown, unknown>,
+    bucketName: string,
+    objectKey: string,
+    ceremonyId?: string
+): Promise<string> => {
+    // Call startMultiPartUpload() Cloud Function.
+    const response: any = await cf({
+        bucketName,
+        objectKey,
+        ceremonyId
+    })
+
+    // Return Multi Part Upload ID.
+    return response.data
+}
+
+/**
+ * Get chunks and signed urls for a multi part upload.
+ * @param cf <HttpsCallable<unknown, unknown>> - the corresponding cloud function.
+ * @param bucketName <string> - the name of the AWS S3 bucket.
+ * @param objectKey <string> - the identifier of the object.
+ * @param filePath <string> - the local path where the file to be uploaded is located.
+ * @param uploadId <string> - the multi part upload unique identifier.
+ * @param expirationInSeconds <number> - the pre signed url expiration in seconds.
+ * @param ceremonyId <string> - the identifier of the ceremony.
+ * @returns Promise<Array, Array>
+*/
+const getChunksAndPreSignedUrls = async (
+    cf: HttpsCallable<unknown, unknown>,
+    bucketName: string,
+    objectKey: string,
+    filePath: string,
+    uploadId: string,
+    expirationInSeconds: number,
+    ceremonyId?: string
+): Promise<Array<ChunkWithUrl>> => {
+    // Configuration checks.
+    if (!process.env.CONFIG_STREAM_CHUNK_SIZE_IN_MB) throw new Error ('Error') 
```

Comment:
Give a short explanation for this error. 

Bonus: use a code (e.g., `throw new Error('S3-001: short explanation here')`).
Hint: in this case, this is not an AWS related error, it's about configs :P